docs/chapters/04_features.md

```diff
@@ -773,6 +773,36 @@ export class GetProductsCount {
 
 
 
+#### Using pagination
+
+The current Booster GraphQL API includes an alternative type for your read models that stands for `List{"your-read-model-name"}`, which includes the functionality to work with pagination.
```

Comment:
I think it is better to say that the `List` queries are the official way to filter read models, and the other way is deprecated and will be deleted in future versions